### PR TITLE
[Web] Add Pipeline Example for Web ML APIs

### PR DESCRIPTION
**Pull request by again4you**
This patch newly adds the Tizen ML Web API example. This app classifies
images and shows the label after the inference.

- Model: MobileNet v1 tf-lite version
- APIs: Web ML Pipeline API including appsrc and appsink

Change-Id: I83ee6defa371ec8ac6e35d77ec5c5526a4f77141
Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: Tizen.web/ImageClassificationPipeline/js/main.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+/**
+ * Get the jpeg image path
+ * @returns image path
+ */
+function GetImgPath() {
+    const MAX_IMG_CNT = 2;
+    if (typeof GetImgPath.count === 'undefined') {
+        GetImgPath.count = 0;
+    }
```
Comment (zhoonit) [inline Tizen.web/ImageClassificationPipeline/js/main.js:47]: how about `GetImgPath.count = 0` in L52?

File: Tizen.web/ImageClassificationPipeline/js/main.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    var imgsrc = GetImgPath.count++ % MAX_IMG_CNT;
+    imgsrc = imgsrc.toString().concat(".jpg");
+    return "/res/".concat(imgsrc);
```
Comment (zhoonit) [inline Tizen.web/ImageClassificationPipeline/js/main.js:50]: wonder if it is supported to use template literals

```suggestion
    return `res/${imgsrc}.jpg`
```

Comment (zhoonit) [reply Tizen.web/ImageClassificationPipeline/js/main.js:50]: If this works, it will be alot cleaner for the pipeline description as well :)

**Review by myungjoo — APPROVED** [review]
LGTM.

In the future, we'll need to consolidate model files.
We already have mobilenet_v1_1.0_224_quant.tflite file in this git repo.
We need to create a model directory and use symlink of them in apps.

**Review by zhoonit — APPROVED** [review]
LGTM!

**Pull request merged**